src/components/image-with-caption/image-with-caption.vue

```diff
@@ -26,7 +26,8 @@ defineProps<{
     height: number,
     url: string,
     alt?: string,
-  }
+    sizes: string,
```

Comment:
`sizes` now is a required prop, but it's not passed in the `gallery-block` component.

Reply:
Well spotted, missed that one somehow. 👍 
Just changed it to use dato-image since it doesn't have a caption.